src/API/src/export/utils/occurrenceMapper.ts

```diff
@@ -0,0 +1,179 @@
+      'recordedSpecies_species_reference_v_data',
+    ],
+    ...itemsArray.map((item) => [
+      item.id,
+      item.month_start,
+      item.year_start,
+      item.month_end,
+      item.year_end,
+      item.dec_id,
+      item.dec_check,
+      item.map_check,
+      item.vector_notes,
+      item.site_id,
+      item.site_country,
+      item.site_name,
+      item.site_map_site,
+      item.site_location_type,
+      item.site_location_coordinates_0,
+      item.site_location_coordinates_1,
+      item.site_latitude,
+      item.site_longitude,
+      item.site_georef_source,
+      item.site_site_notes,
+      item.site_gaul_code,
+      item.site_admin_level,
+      item.site_georef_notes,
+      item.site_admin_1,
+      item.site_admin_2,
+      item.site_admin_3,
+      item.site_admin_2_id,
+      item.site_location_2_type,
+      item.site_location_2_coordinates_0,
+      item.site_location_2_coordinates_1,
+      item.site_latitude_2,
+      item.site_longitude_2,
+      item.site_latlong_source,
+      item.site_good_guess,
+      item.site_bad_guess,
+      item.site_rural_urban,
+      item.site_is_forest,
+      item.site_is_rice,
+      item.site_area_type,
+      item.sample_id,
+      item.sample_mossamp_tech_1,
+      item.sample_n_1,
+      item.sample_mossamp_tech_2,
+      item.sample_n_2,
+      item.sample_mossamp_tech_3,
+      item.sample_n_3,
+      item.sample_mossamp_tech_4,
+      item.sample_n_4,
+      item.sample_n_all,
+      item.sample_control,
+      item.sample_control_type,
+      item.reference_id,
+      item.reference_author,
+      item.reference_article_title,
+      item.reference_journal_title,
+      item.reference_citation,
+      item.reference_year,
+      item.reference_published,
+      item.reference_report_type,
+      item.reference_v_data,
+      item.recordedSpecies_id,
+      item.recordedSpecies_ss_sl,
+      item.recordedSpecies_assi,
+      item.recordedSpecies_assi_notes,
+      item.recordedSpecies_id_method_1,
+      item.recordedSpecies_id_method_2,
+      item.recordedSpecies_id_method_3,
+      item.recordedSpecies_species_id,
+      item.recordedSpecies_species_subgenus,
+      item.recordedSpecies_species_series,
+      item.recordedSpecies_species_section,
+      item.recordedSpecies_species_complex,
+      item.recordedSpecies_species_species,
+      item.recordedSpecies_species_species_author,
+      item.recordedSpecies_species_year,
+      item.recordedSpecies_species_reference_id,
+      item.recordedSpecies_species_reference_author,
+      item.recordedSpecies_species_reference_article_title,
+      item.recordedSpecies_species_reference_journal_title,
+      item.recordedSpecies_species_reference_citation,
+      item.recordedSpecies_species_reference_year,
+      item.recordedSpecies_species_reference_published,
+      item.recordedSpecies_species_reference_report_type,
+      item.recordedSpecies_species_reference_v_data,
+    ]),
```

Comment:
Is this the only way of doing this? It's pretty brittle against any changes to the db - we'd have to change this file too. Can we do it in a more 'flexible' way?